### DISCUSSION TITLE
Is there a way to force update user permissions or force loggout of specific user that currently have a session active?

### DISCUSSION THREAD

**Discussion by JStafford00** [OrchardCMS/OrchardCore#13187] [Q&A] [answered]
So, say I update a certain user's role, or disable a user's account. Is there a way that OC can force update the user's active session, or force them to logout? I've noticed that even after changing a user role or  something similar that they are still able to view/modify content that they no longer have permission to view/modify as long as their current session is still active.

**Comment by lampersky** [comment, marked as answer]
you can force logout user using this code:

```
await userManager.UpdateSecurityStampAsync(user);
await signInManager.SignOutAsync();
```

also you need to update your configuration (Startup.cs):

```
services.Configure<SecurityStampValidatorOptions>(options =>
{
    options.ValidationInterval = TimeSpan.Zero;
});
```

**Reply by JStafford00** [reply]
Is there a way for an admin to force logout another user?

**Reply by lampersky** [reply]
just update the security stamp of the user which should be logged out, if validation interval is set to zero, user will be logged out immediately
